pkg/networkmanager/firewall-client.js

```diff
@@ -309,19 +309,19 @@ function fetchZoneInfos(zones) {
             return info;
         }
         return firewalld_dbus.call('/org/fedoraproject/FirewallD1',
-                                   'org.fedoraproject.FirewallD1',
-                                   'getZoneSettings', [zone])
+                                   'org.fedoraproject.FirewallD1.zone',
+                                   'getZoneSettings2', [zone])
                 .then(reply => {
-                    const [, name, description, , target, services, ports, , , , interfaces, source] = reply[0];
+                    const zoneInfo = reply[0];
```

Comment:
While at it, would you mind cleaning this up to `.then(([zoneInfo]) => ...`? (I.e.structured decomposition). In principle you can do this even one step further with `([{ description, target, services, ... }])`, but it doesn't help that much due to the required `.v` and fallbacks.